spec/presenters/braintree_presenter_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe "BraintreePresenter"
```

Comment:
These tests don't even seem to work as they are not valid syntax. Was there a bad merge or something?